manifests-maistra/2.3.0/maistraoperator.v2.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,794 @@
+                olm.relatedImage.v2_3.proxyv2: quay.io/maistra/proxyv2-ubi8:2.3.0
+                olm.relatedImage.v2_3.rls: quay.io/maistra/ratelimit-ubi8:2.3.0
+            spec:
+              serviceAccountName: istio-operator
+              containers:
+                - name: istio-operator
+                  image: quay.io/maistra/istio-ubi8-operator:2.3.0
+                  ports:
+                    - containerPort: 11999
+                      name: validation
+                    - containerPort: 60000
+                      name: metrics
+                  command:
+                    - istio-operator
+                    - --config
+                    - /etc/operator/olm/config.properties
+                  imagePullPolicy: Always
+                  env:
+                    - name: WATCH_NAMESPACE
+                      value: ""
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: OPERATOR_NAME
+                      value: istio-operator
+                      #        - name: ISTIO_CNI_IMAGE_PULL_SECRET
+                      #          value: name-of-secret
+                  volumeMounts:
+                    - name: operator-olm-config
+                      mountPath: /etc/operator/olm
+                      readOnly: true
+                    - name: webhook-tls-volume
+                      readOnly: true
+                      mountPath: /tmp/k8s-webhook-server/serving-certs
+                    - name: smcp-templates
+                      readOnly: true
+                      mountPath: /usr/local/share/istio-operator/templates/
+              volumes:
+                - name: operator-olm-config
+                  downwardAPI:
+                    defaultMode: 420
+                    items:
+                      - fieldRef:
+                          fieldPath: metadata.annotations
+                        path: config.properties
+                - name: webhook-tls-volume
+                  secret:
+                    secretName: maistra-operator-serving-cert
+                    optional: true # this won't be available until service-ca creates the secret
+                - name: smcp-templates
+                  configMap:
+                    name: smcp-templates
+                    optional: true
+  customresourcedefinitions:
+    owned:
+    - name: servicemeshcontrolplanes.maistra.io
+      version: v2
+      kind: ServiceMeshControlPlane
+      displayName: Istio Service Mesh Control Plane
+      description: An Istio control plane installation
+      specDescriptors:
+      - displayName: Control Plane Version
+        description: Specify the version of the control plane you want to install
+        path: version
+        x-descriptors:
+          - 'urn:alm:descriptor:com.tectonic.ui:fieldGroup:General'
+          - 'urn:alm:descriptor:com.tectonic.ui:select:v2.1'
```

Comment:
Need to add entries for v2.2 and v2.3 above this line. We should also fix this in v2.2.

Reply:
Done